src/app/collection/pages/collection-ncyte/collection-ncyte.component.html

```diff
@@ -0,0 +1,10 @@
+<div>
+  <clark-header-ncyte></clark-header-ncyte>
+  <clark-feature></clark-feature> 
```

Comment:
The clark-feature should also take in a collectionName so we can use it for other collection pages.

Reply:
what would all the collection pages need from feature?

Reply:
Feature is where the request is being made to get the featured learning objects right? Each page will need to get their respective collections featured learning objects.

Reply:
Theres actually no request being done in feature, the request is being done in the collection learning object list. Should i move the request to the feature component?

Reply:
No but the collection learning object list is a child of the feature component so the feature component shouldn't have a hard coded value for a collection it should be sent down from the respective collection pages.

Reply:
Wouldnt the feature component be slightly different for each page though? Like the background color and such.

Reply:
Yes, but we can refactor later to take in variables for colors and such.